backend_new/src/services/listing.service.ts

```diff
@@ -738,7 +738,7 @@ export class ListingService implements OnModuleInit {
         jurisId: rawListing.jurisdictions.id,
       });
     }
-
+    await this.cachePurge(ListingsStatusEnum.closed, dto.status, rawListing.id);
```

Comment:
Nit: I understand what this is doing but I think an update to the cachePurge function would be helpful to make this use case more readable. Passing closed as the storedListingStatus is a bit confusing since this is a brand new listing. I'd prefer to see this passed as undefined since that seems like the most accurate description of a new listing and then have cachePurge handle that case

Reply:
Hey @ColinBuyck agree, went ahead and made the update for ya

once tests pass will merge ty!